src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -59,6 +59,7 @@ public enum QuicError
         ConnectionIdle = 10,
         ProtocolError = 11,
         OperationAborted = 12,
+        OSDoesNotSupportIPv6 = 13
```

Comment:
We will need API approval to modify public enum

Reply:
should I open a separate issue to suggest this change and add keep in this PR just the msquic new status code ?

Reply:
Open a separate issue please, you can get inspiration from https://github.com/dotnet/runtime/issues/74755.